writing-visualising/dashboards.qmd

```diff
@@ -513,6 +513,39 @@ An example of a basic shiny test to check that your app loads up correctly can b
 
 ---
 
+#### Migration to shinytest2
+
+---
+
+The shinytest package has been superceded by the [shinytest2 package](https://rstudio.github.io/shinytest2/). If you already have shinytest UI tests in place for your dashboard, there is a function available to migrate these tests for you so that you do not have to rewrite them all yourself. 
```

Comment:
Spelling - supersede rather than supercede, and I enjoy this mini description on google that I found when I questioned myself on it:

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/d0bb1d8f-5e8c-420c-b6ab-18e7c305023b)

Reply:
I love this 😆